File: Exercism/csharp/raindrops/Raindrops.cs
using System;

public static class Raindrops
{
	public static string Convert(int n)
	{
		string rain = "";

		if(n % 3 == 0)
		{
			rain += "Pling";
		}
		if(n % 5 == 0)
		{
			rain += "Plang";
		}
		if(n % 7 == 0)
		{
			rain += "Plong";
		}
		if(n % 3 != 0 && n % 5 != 0 && n % 7 != 0)
		{
			rain += n.ToString();
		}

		return rain;
	}
}

File: Exercism/typescript/raindrops/raindrops.ts
export function convert(_n: number): string {
  let rain: string = ""

  if(_n % 3 == 0)
  {
    rain += "Pling"
  }

  if(_n % 5 == 0)
  {
    rain += "Plang"
  }

  if(_n % 7 == 0)
  {
    rain += "Plong"
  }

  if(_n % 3 != 0 && _n % 5 != 0 && _n % 7 != 0)
  {
    rain += _n
  }

  return rain
}
